ui/pages/bridge/hooks/useGasIncluded7702.ts

```diff
@@ -42,6 +43,9 @@ export function useGasIncluded7702({
 }: UseGasIncluded7702Params): boolean {
   const [isGasIncluded7702Supported, setIsGasIncluded7702Supported] =
     useState(false);
+  const smartTransactionsEnabled = useSelector((state) =>
```

Comment:
nit: can you please rename it to `isSmartTransaction`, so it would follow the naming from `getIsSmartTransaction`?